pkgs/applications/graphics/focus-stack/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, which
+, ronn
+, opencv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "focus-stack";
+  version = "1.4";
+
+  src = fetchFromGitHub {
+    owner = "PetteriAimonen";
+    repo = "focus-stack";
+    rev = version;
+    hash = "sha256-SoECgBMjWI+n7H6p3hf8J5E9UCLHGiiz5WAsEEioJsU=";
+  };
+
+  nativeBuildInputs = [ pkg-config which ronn ];
+  buildInputs = [ opencv ];
+
+  makeFlags = [ "DESTDIR=$(out)" "prefix=" ];
```

Comment:
See https://github.com/NixOS/nixpkgs/issues/65718 for the correct usage of DESTDIR and prefix.